### ISSUE TITLE
[Accessibility] Conduct mapping exercise to identify poster alternatives

### ISSUE THREAD

**Issue by niradatta1** [department-of-veterans-affairs/va.gov-team#31485]
### Background
Print posters are inaccessible to people who are blind and those living with low-vision impairments. Challenges include: 

- Knowing there is a poster present in the first place
- Knowing that one can scan it

How might we offer  multiple alternatives to the first-touch point in the check in experience --- the poster ---  to make it more accessible?

[Link to the early list of possible ideas and meeting notes with Marth/Josh/Angela here. ](https://docs.google.com/document/d/1i1sqKzU_t_L8V-1pdNcG8bxUnL9bNwu2GVSFEADNp-4/edit)
Summary:
- Our goal is to identify a combination of alternatives to the poster which will meet the needs of veterans living with different types of disabilities.
- Next step is for Ya-ching and I to map out possible solutions according to accessibility use cases/scenarios (and possibly journey mapping), and then formulate a recommendation.
- We'll be collaborating with Josh Kim, Martha and Angela
- We'll be sure to take the recommendation to the team to discuss feasibility / prioritization and what makes sense for near-term and long-term.

We recognize that there is no “one-size-fits-all” solution. We’ll need to propose a variety of options (some technology-based, some service design based) to address a variety of disabilities.  To do this, we'll conduct the following tasks: 
- [x]  Map out possible solutions according to accessibility use cases + user needs
- [x]  Schedule time with Josh, Martha, Angela

Note: this may also include generation a journey map based on different accessibility scenarios / use cases. 

### Acceptance Criteria
Link to draft of the mapping exercise of possible alternatives for each scenario  (Google Sheets / Google Docs, per requested by accessibility team), with recommendation ready to discuss with UX/Product team for feedback.

**Comment by julieadair1** [comment]
Note* Refer to Martha's slack about the legal precedence re: Rite Aid and Dominos

**Comment by julieadair1** [comment]
@jparker0425 what epic should this be attached to?

**Comment by kristenmcconnell** [comment]
@Nira and @ytsaoca In part 2, we will document the recommendations, then share with check-in team. The conversations with accessibility and draft map have been completed.
